### DISCUSSION TITLE
[BUG] setTitle no longer works in v6.4 and v6.4.1

### DISCUSSION THREAD

**Discussion by gaspesla** [elgatosf/elgato#74] [Help] [answered]
We are seeing a change in the behavior of our plugin upon updating to SD v6.4 and v6.4.1 **on Mac**:

Our plugin has been around for years and has thousands of users, and **setTitle** has always reliably set the title of keys.
**Now it no longer seems to do anything at all.**

I have observed that it starts working again if I remove the **Title** attributes from the **States** array in that key's manifest entry.
However I'd like to keep that in there if possible, as the title in the manifest was serving a a "default title" that would display before the plugin had a change to set it programmatically. 

Everything else seems to be working fine, including **setImage** and **setState**.

Can someone please let me know if this was an intentional change and/or something that will be fixed?

Thanks, Glen

**Comment by GeekyEggo** [comment]
Hey @gaspesla, sadly it's come to our attention that as part of Stream Deck 6.4 a regression was introduced whereby `setTitle` ceases to work if a default `Title` is defined for an action within the manifest. This is not expected behaviour, and the Stream Deck team are working on fix, and our current plan is to apply this as a hot-fix that will automatically be rolled out to users; I will provide updates here as I know more. My apologies for the inconvenience caused to you and your users.

**Comment by GeekyEggo** [comment, marked as answer]
This has now be fixed as part of [Stream Deck 6.5](https://www.elgato.com/us/en/s/downloads), released today! 🎉
